common.go

```diff
@@ -115,6 +115,7 @@ func applyPathModifiers(path string, modifiers []string) string {
 	substPtn := regexp.MustCompile("s\\/([^\\/]+)\\/([^\\/]*)\\/")
 	trimEndPtn := regexp.MustCompile("%(.*)")
 	basenamePtn := regexp.MustCompile(`.*\/`)
+	dirnamePth := regexp.MustCompile(`\/[^\/]*$`)
```

Comment:
There is a small typo here: `h` should be `n` in `dirnamePth`, so `dirnamePtn`.

Reply:
Thank you for pointing it out. I fixed this typo.